core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -195,7 +195,7 @@ $(document).ready(function() {
         }
 
         $(selector).DataTable(conf);
-        $('#hisotry-summary [data-toggle="tooltip"]').tooltip();
+        $('#history-summary-table [data-toggle="tooltip"]').tooltip();
```

Comment:
I have fixed the spelling error. The test result is OK.  But historypage-template.html still need to be modified. Thanks.